engine/runner/src/main/java/org/enso/runner/JavaFinder.java

```diff
@@ -89,6 +93,9 @@ private static Path findJavaExecutableInDistributionRuntimes() {
           versionUsedForBuild);
       return newerRuntime.get().javaExecutable();
     }
+    logger.trace("No appropriate runtime found in the distribution.");
+    logger.trace("graalVersionManager.getAllRuntimes() = {}", graalVersionManager.getAllRuntimes());
+    logger.trace("Paths of distributionManager = {}", distributionManager.paths());
```

Comment:
It's better to use a single log line. Multiple lines can be mixed up with other logs making the output less clear